static/app/gettingStartedDocs/node/connect.tsx

```diff
@@ -68,8 +69,14 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)].",
-            {code1: <code />, code2: <code />}
+            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:different installation methods in our docs].",
```

Comment:
```suggestion
            "Make sure to import [code1:instrument.js/mjs] at the top of your file. Set up the error handler after all controllers and before any other error middleware. This setup is typically done in your application's entry point file, which is usually [code2:index.(js|ts)]. If you are unable to import an external file, read about the [docs:alternate installation methods in our docs].",
```

Reply:
I think "different" is more fitting than "alternate" here. But I could also write: "read about alternative installation methods in our docs" 🤔

Reply:
I think "read about alternative installation methods" reads slightly nicer, we also call them alternate installation methods in the docs, but no strong feelings on it.